package.json

```diff
@@ -19,7 +20,8 @@
     "release:firefox": "web-ext build --source-dir dist/src --overwrite-dest --artifacts-dir dist/",
     "release": "npm run release:chrome && npm run release:firefox",
     "presign": "npm run release",
-    "sign": "web-ext sign --source-dir dist/src --artifacts-dir dist/ --id \"{d403ee9c-3bd2-41d3-b1e9-27698babf097}\""
+    "sign": "web-ext sign --source-dir dist/src --artifacts-dir dist/ --id \"{d403ee9c-3bd2-41d3-b1e9-27698babf097}\"",
+    "watch": "chokidar \"src/*.js\" -c \"npm run build:js\""
```

Comment:
Should this watch the whole `src/` directory and run `npm run build` instead so it will pack css file changes too?

Reply:
There's no need to bring new dependency, rollup already has a [watch option](https://rollupjs.org/guide/en/#-w--watch) in place. Maybe use that instead?

Reply:
@cbotsikas yes, you are correct. I updated it.
@tiratatp I think it not necessary to run 'build' command since it includes manifest task. I agree with also watching on css file. However, I looked on rollup and didn't see any direct way to config it watches on css file then run npm command. I found nodemon can watch on multiple directories. Is it alright?